common/src/stack/command/stack/commands/remove/host/interface/plugin_cleanup.py

```diff
@@ -0,0 +1,71 @@
+# @copyright@
+# Copyright (c) 2006 - 2018 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+#
+# @rocks@
+# Copyright (c) 2000 - 2010 The Regents of the University of California
+# All rights reserved. Rocks(r) v5.4 www.rocksclusters.org
+# https://github.com/Teradata/stacki/blob/master/LICENSE-ROCKS.txt
+# @rocks@
+
+import stack.commands
+
+
+class Plugin(stack.commands.Plugin):
+
+	def provides(self):
+		return 'cleanup'
+
+	def requires(self):
+		return [ 'HEAD' ]
+		
+	def removerow(self, interfaceid):
+		self.owner.db.execute("""delete from networks where id = '%s'""" % interfaceid)
+
+	def ifclean(self, interfaceid, interface):
+		#
+		# - turn off the interface
+		# - remove the ifcfg file (if necessary)
+		# - remove it from the live configuration
+		#
+		import subprocess
```

Comment:
Unless you have a good reason not to, these imports should go at the top of the file.